packages/framework/src/Model/Transport/Transport.php

```diff
@@ -242,4 +251,51 @@ protected function createTranslation()
     {
         return new TransportTranslation();
     }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Payment\Payment $payment
+     */
+    public function addPayment(Payment $payment)
+    {
+        if (!$this->payments->contains($payment)) {
+            $this->payments->add($payment);
+            $payment->addTransport($this);
+        }
+    }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Payment\Payment[] $payments
+     */
+    public function setPayments(array $payments)
```

Comment:
If I have payment `$pay` and transport `$trans` linked together and call `$trans->setTransports([])` I should remove the link. Payment `$pay` will still have the link to the transport `$trans`.